packages/test/src/workflows/sdk-internal-patch-after.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * "Before" version of a workflow used to reproduce an issue with SDK internal patch,
+ * where the user modifies his code so that the SDK internal patch no longer gets generated.
+ *
+ * @module
+ */
+import { condition, sleep } from '@temporalio/workflow';
+
+/**
+ * Patches the workflow from ./patch-and-condition-pre-patch, adds a patched statement inside a condition.
+ */
+export async function sdkInternalPatch(): Promise<void> {
+  await condition(() => false, 10);
```

Comment:
I think we just live with the fact that changing condition duration from 0 to anything else is a non deterministic change due to this internal patch unless we want to generate another patch that wraps both cases.

Let's wait a bit longer to have a standard way to do internal SDK patches.

Reply:
I agree. Closing.